servers/keycloak/src/main/docker/keycloak-realm/ups-realm-sample.json

```diff
@@ -0,0 +1,69 @@
+{
+    "realm": "aerogear",
+    "enabled": true,
+    "accessTokenLifespan": 60,
+    "accessCodeLifespan": 60,
+    "accessCodeLifespanUserAction": 300,
+    "ssoSessionIdleTimeout": 600,
+    "ssoSessionMaxLifespan": 36000,
+    "sslRequired": "external",
+    "registrationAllowed": false,
+    "social": false,
+    "updateProfileOnInitialSocialLogin": false,
+    "requiredCredentials": [ "password" ],
+    "defaultRoles": [ "developer" ],
+    "users" : [
+        {
+            "username" : "admin",
+            "enabled": true,
+            "credentials" : [
+                { "type" : "password",
+                    "value" : "123" }

```

Comment:
[Question] Is that standard password and account for keycloak? Maybe we need to put some minimal documentation for this new change?

Reply:
no, this is the ups admin, and its initial default password. we have this since ages